includes/admin/class-sensei-onboarding.php

```diff
@@ -73,7 +80,7 @@ function( $classes ) {
 	public function admin_menu() {
 		if ( current_user_can( 'manage_sensei' ) ) {
 			add_submenu_page(
```

Comment:
Is there a compelling reason why we're hiding this menu instead of removing it altogether? I'm not seeing why we would want to keep it.

Reply:
This is needed for the admin page to exists and be accessible. Not sure if there is an alternative way of adding it, I haven't found any previously.

Reply:
OK, was curious.